selfservice/flow/login/hook.go

```diff
@@ -23,7 +23,7 @@ type (
 	}
 
 	PostHookExecutor interface {
-		ExecuteLoginPostHook(w http.ResponseWriter, r *http.Request, g node.Group, a *Flow, s *session.Session) error
+		ExecuteLoginPostHook(w http.ResponseWriter, r *http.Request, g node.UiNodeGroup, a *Flow, s *session.Session) error
```

Comment:
Is the `g` parameter actually used? I've checked this diff https://github.com/Wikia/kratos/compare/v0.10.1...PLATFORM-7527 and it looks like the g parameter isn't there anymore.